### ISSUE TITLE
react-native version 0.69.0 support 

### ISSUE THREAD

**Issue by atombreak** [ide/react-native-button#95]
Hi! 👋 
      
Firstly, thanks for your work on this project! 🙂

Today I used [patch-package](https://github.com/ds300/patch-package) to patch `react-native-button@2.4.0` for the project I'm working on.

Here is the diff that solved my problem:

```diff
diff --git a/node_modules/react-native-button/Button.js b/node_modules/react-native-button/Button.js
index fb7cf46..3609e38 100644
--- a/node_modules/react-native-button/Button.js
+++ b/node_modules/react-native-button/Button.js
@@ -1,4 +1,5 @@
 import PropTypes from 'prop-types';
+import {TextPropTypes} from 'deprecated-react-native-prop-types'
 import React, { Component } from 'react';
 import {
   StyleSheet,
@@ -16,12 +17,12 @@ export default class Button extends Component {
   static propTypes = {
     ...TouchableOpacity.propTypes,
     accessibilityLabel: PropTypes.string,
-    allowFontScaling: Text.propTypes.allowFontScaling,
+    allowFontScaling: TextPropTypes.allowFontScaling,
     containerStyle: ViewPropTypes.style,
     disabledContainerStyle: ViewPropTypes.style,
     disabled: PropTypes.bool,
-    style: Text.propTypes.style,
-    styleDisabled: Text.propTypes.style,
+    style: TextPropTypes.style,
+    styleDisabled: TextPropTypes.style,
     childGroupStyle: ViewPropTypes.style,
   };
 
```

<em>This issue body was [partially generated by patch-package](https://github.com/ds300/patch-package/issues/296).</em>

**Comment by ide** [comment]
Please send a PR with the text "Closes #95" to address this.

**Comment by ide** [comment]
Addressed by https://github.com/ide/react-native-button/pull/96
